src/client/index.js

```diff
@@ -2,43 +2,34 @@
 import React from "react"
 import ReactDOM from "react-dom"
 import Router, { browserHistory } from "react-router"
-import { Provider } from "react-redux"
+import { Provider as ReduxContextProvider } from "react-redux"
 
-import fetchJSON from "../fetchJSON"
-import MetadataProvider from "../MetadataProvider"
+import StatinamicContextProvider from "../ContextProvider"
 
 export default function statinamic({
+  layouts,
   metadata,
   routes,
   store,
 }) {
   let devtools = false
-  if (__DEVTOOLS__) {
+  if (process.env.REDUX_DEVTOOLS) {
```

Comment:
This line https://github.com/MoOx/statinamic/blob/babel-node-loaders/src/redux/createStore/index.js#L33 should be 

```
process.env.REDUX_DEVTOOLS && 
typeof window !== "undefined"
```

because on line 38 of the same file. We are using window object which is undefined in static build.

Anyway, just run `REDUX_DEVTOOLS=true npm start` and you will see a bunch of error